qiskit/circuit/quantumcircuit.py

```diff
@@ -904,8 +907,9 @@ class QuantumCircuit:
 
     If a :class:`QuantumCircuit` has been scheduled as part of a transpilation pipeline, the timing
     information for individual qubits can be accessed.  The whole-circuit timing information is
-    available through the :attr:`duration`, :attr:`unit` and :attr:`op_start_times` attributes.
+    available through the :meth:`estimate_duration` method and :attr:`op_start_times` attribute.
```

Comment:
Are we leaving `op_start_times` as a valid attribute of `QuantumCircuit` post deprecation?

Reply:
I didn't think that we deprecated it as part of 1.3.0. There are also a bunch of associated methods with it too. We can deprecate it for 1.4.0 as falls into the same category as duration though and we should probably remove it from the circuit object. But we can look at that in a separate pr.

Reply:
I looked into doing this and don't think we should for 2.0. The timeline visualizer depends on this data and methods. I need to figure out a more complete mechanism to rewrite the drawer without this, and rushing it in the week of 1.4.0 doesn't seem ideal. We should update the docstrings in 2.0 though to make it more explicit that this is an estimate etc, like what this method does for the duration.

Reply:
I added an issue to keep track of the docstring update: https://github.com/Qiskit/qiskit/issues/13880 it's small but I am afraid that without it we would forget to do it.